docs/user/getting-started.md

```diff
@@ -132,7 +132,7 @@ Install a released version of the Flight Control Service into the cluster by run
 $ helm upgrade --install --version=<version-to-install> \
     --namespace flightctl --create-namespace \
     flightctl oci://quay.io/flightctl/charts/flightctl \
-    --set global.target=acm
+    --values deploy/helm/flightctl/values.acm.yaml
```

Comment:
this will work fine when you have the helm chart locally, but running it from oci registry, the values.acm.yaml wont exist on disk.

In the getting-started doc, we should instruct users to:
 1. create the values.acm.yaml with correct content
 2. use `--values ./values.acm.yaml`

Reply:
Done